Quadriga/src/main/webapp/WEB-INF/tiles/layouts/skeleton_public_sites.jsp

```diff
@@ -105,21 +110,39 @@
 </head>
 <body>
 	<tiles:importAttribute name="currentPage" scope="request" />
-	
-	 <div class="container-fluid">
-      <div class="header clearfix">
-        <nav>
-          <ul class="nav pull-right">
-            <tiles:insertAttribute name="navigation" />
-          </ul>
-        </nav>
-        <h3 class="text-muted"><a class="navbar-brand" href="${pageContext.servletContext.contextPath}/sites"><img
-                    src="${pageContext.servletContext.contextPath}/resources/bootstrap-theme/assets/img/QuadrigaLogo-grey.png"
-                    height="35px" style="margin-top: -5px;"></a></h3>
-      </div>
 
-		
-		<tiles:insertAttribute name="content" />
+	<div class="container-fluid">
+		<div class="header clearfix">
+			<div class="navbar-collapse collapse navbar-right">
+				<ul class="nav navbar-nav">
+					<tiles:insertAttribute name="navigation" />
+				</ul>
+			</div>
+			<h3 class="text-muted">
+				<a class="navbar-brand"
+					href="${pageContext.servletContext.contextPath}/sites"><img
+					src="${pageContext.servletContext.contextPath}/resources/bootstrap-theme/assets/img/QuadrigaLogo-grey.png"
+					height="35px" style="margin-top: -5px;"></a>
+			</h3>
+		</div>
+
+		<tilesx:useAttribute name="right-navigation" id="rightnavigation" />
+		<div <c:if test="${!empty rightnavigation}"> class="col-md-9"</c:if>
+			style="text-align: center">
+			<div class="content">
+				<!-- Content -->
+				<tiles:insertAttribute name="content" />
+				<!-- /Content -->
+			</div>
+		</div>
+
+		<div class="col-md-3" style="text-align: center">
+			<!-- Sidebar -->
+			<!-- Recent Posts -->
+			<tiles:insertAttribute name="right-navigation" />
+			<!-- /Recent Posts -->
+			<!-- /Sidebar -->
+		</div>
```

Comment:
Is this div always on the page? Even if there is no right navigation? That's not good. Why don't you just use two separate skeletons?